src/java/org/apache/cassandra/io/sstable/format/SortedTableScrubber.java

```diff
@@ -318,7 +318,12 @@ protected boolean tryAppend(DecoratedKey prevKey, DecoratedKey key, SSTableRewri
      */
     private UnfilteredRowIterator getIterator(DecoratedKey key)
     {
-        RowMergingSSTableIterator rowMergingIterator = new RowMergingSSTableIterator(SSTableIdentityIterator.create(sstable, dataFile, key), outputHandler);
+        RowMergingSSTableIterator rowMergingIterator =new RowMergingSSTableIterator(SSTableIdentityIterator.create(sstable,
```

Comment:
Super nit: missing whitespace after equals:
```suggestion
        RowMergingSSTableIterator rowMergingIterator = new RowMergingSSTableIterator(SSTableIdentityIterator.create(sstable,
```

Reply:
Done and committed